Editor/Module Management/ModuleInstaller.cs

```diff
@@ -35,19 +35,25 @@ public static class ModuleInstaller
         private const float TIMEOUT_FOR_MODULE_INSTALLATION = 20f;
         private const string AVATAR_LOADER_SUBSTRING = "avatarloader";
 
+
         static ModuleInstaller()
         {
+
             Events.registeredPackages += OnRegisteredPackages;
             Events.registeringPackages += OnRegisteringPackages;
         }
 
+
         /// <summary>
         ///     Called when a package is added, removed or changed.
         /// </summary>
         /// <param name="args">Describes the <c>PackageInfo</c> entries of packages that have just been registered.</param>
         private static void OnRegisteredPackages(PackageRegistrationEventArgs args)
         {
             Events.registeredPackages -= OnRegisteredPackages;
+#if RPM_DEVELOPMENT
```

Comment:
What determines this being set to true? Is it effectively an env var based on the build?

Reply:
Well kinda. 
It's a scripting define symbol that can be added from the project settings. But by default it won't be there. 
Currently it serves the purpose of :
- preventing the auto installer from running (used for Unity-SDK or anybody wanting to install the package into Assets folder for development purposes)
- and of course it also determines which environment the analytics gets sent to, more so for us internally so we don't add lots of garbage events during development 


![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/6229f5e2-d507-4714-9e3a-32e9b08a5e38)